phoenicis-tools/src/main/java/org/phoenicis/tools/files/FileUtilities.java

```diff
@@ -162,7 +166,8 @@ public void remove(String path) throws IOException {
         final File fileToDelete = new File(path);
 
         if (!fileToDelete.exists()) {
-            throw new IllegalArgumentException(String.format("Path \"%s\" does not exist", path));
+            LOGGER.debug(String.format("Cannot remove file or directory: path \"%s\" does not exist", path));
+            return;
         }
 
         assertInDirectory(fileToDelete);
```

Comment:
I think the assertion should then be moved before `if (!fileToDelete.exists()) { .. }`

Reply:
Why should it matter?

Reply:
The order seems wrong like this. Let us say the script wants to execute `remove("/home/<your/home>/important-file");` then I wouldn't expect a debug message that says the file does not exist but an error message that tells me that the script tries to access files it has no access to

Reply:
If you want we can handle this in another PR. I just want to talk about this now that it is still fresh on my mind

Reply:
Ok.